pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -0,0 +1,142 @@
+package certificate
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestCertManager(t *testing.T) {
+	test.NewTest(t).Id("T38").Groups(test.Full, test.ARM, test.InterOp).Run(func(t test.TestHelper) {
+
+		ns := "foo"
+		cert := "cert-manager"
+		t.Cleanup(func() {
+			oc.DeleteNamespace(t, cert)
+			oc.RecreateNamespace(t, ns)
+		})
+
+		t.LogStep("uninstall the SMCP")
+		oc.RecreateNamespace(t, meshNamespace)
+
+		t.LogStep("Add jetstack repo to helm")
+		shell.Execute(t, `helm repo add jetstack https://charts.jetstack.io`)
+
+		t.LogStep("Install cert-manager")
+		shell.Execute(t,
+			fmt.Sprintf(`helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.11.0 --set installCRDs=true`),
```

Comment:
```suggestion
			`helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.11.0 --set installCRDs=true`,
```

Reply:
oh, we don't require the "fmt.Sprintf()" 

Thats nice